tasks/spec.rake

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'open3'
+
+def execute_and_stream(cmd)
+  Open3.popen3(cmd) do |_stdin, stdout, stderr, _thread|
+    until (raw_line = stdout.getc).nil?
+      print raw_line.to_s
+    end
+
+    until (raw_line = stderr.getc).nil?
+      print raw_line.to_s
+    end
+  end
+end
+
+desc 'Run rspec test in sequential order'
+task :spec do
+  execute_and_stream('bundle exec rspec')
+end
```

Comment:
Instead of relying in open3, could we rely on rspec mechanism of task def?
https://relishapp.com/rspec/rspec-core/docs/command-line/rake-task
we can decide on which files and flags are used on each rake task.

Reply:
Great suggestion, made the proposed changes.